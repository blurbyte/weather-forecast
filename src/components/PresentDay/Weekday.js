import React from 'react';
import styled from 'styled-components';

import Text from '../Text';
import BaseRow from '../Row';
import Label from './Label';

const Row = styled(BaseRow)`
  margin: 1rem 0;
`;

const Weekday = () => (
  <Row>
    <Text big>
      <Label>Today</Label> Wednesday
    </Text>
  </Row>
);

export default Weekday;
