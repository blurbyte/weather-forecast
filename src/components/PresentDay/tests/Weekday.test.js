import React from 'react';
import renderer from 'react-test-renderer';

import Weekday from '../Weekday';

jest.mock('../Label', () => 'Label');

test('renders correctly', () => {
  const component = renderer.create(<Weekday timestamp={1526714160} />);
  expect(component.toJSON()).toMatchSnapshot();
});
