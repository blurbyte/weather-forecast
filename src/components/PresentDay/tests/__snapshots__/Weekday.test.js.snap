// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-htpNat hKggGN sc-bwzfXH kRemwO"
>
  <p
    className="sc-bdVaJa cRsIjw"
  >
    <Label>
      Today
    </Label>
     
    Sunday
  </p>
</div>
`;
